components/dashboard/src/settings/Preferences.tsx

```diff
@@ -153,6 +161,12 @@ export default function Preferences() {
                     </div>
                 </SelectableCard>
             </div>
+            <h3 className="mt-12">Dotfiles <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Beta</PillLabel></h3>
+            <p className="text-base text-gray-500 dark:text-gray-400">Customise every workspace using dotfiles. Add a repo below which gets cloned and installed during workspace startup.</p>
+            <div className="mt-4">
```

Comment:
**issue(non-blocking)**: The max width of the text input element could be limited as also @jldec mentioned in https://github.com/gitpod-io/gitpod/pull/7337#issuecomment-1002740006. I'd suggest to go with `max-w-md`.

```suggestion
            <div className="mt-4 max-w-md">
```